build/mobilenet-v1/build.py

```diff
@@ -43,6 +46,20 @@
 )
 
 model_name = "mobilenetv1-w4a4"
+model_file = "models/%s_pre_post_tidy.onnx" % model_name
+
+verif_en = os.getenv("VERIFICATION_EN", "0")
+
+
+def custom_step_update_model(model, cfg):
```

Comment:
This function shouldn't be necessary anymore, please check if it can be removed.

Reply:
Resolved in [c3023fb](https://github.com/Xilinx/finn-examples/pull/109/commits/c3023fbd27e2710e9d3c4367d92c75653a04ee30)